packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -857,14 +861,19 @@ class IrisGridTableModel extends IrisGridModel {
   }
 
   get formatColumns() {
-    return this.formatColumnList;
+    return this.formatColumnList ?? [];
   }
 
   set formatColumns(formatColumns) {
-    if (deepEqual(formatColumns, this.formatColumnList)) {
-      log.debug('Ignore same formatColumns');
+    if (this.formatColumnList === null && formatColumns.length === 0) {
+      log.debug('Ignore empty initial formatColumns');
+      this.formatColumnList = formatColumns;
       return;
     }
+    if (this.formatColumnList === formatColumns) {
+      log.debug('Ignore initial empty formatColumns');
+      this.formatColumnList = formatColumns;
+    }
```

Comment:
What's going on here? If it's the same ... shouldn't need to set it, and can just return? Maybe a bad copy/paste from the if block above?